### PR TITLE
Update .update for luma 10.0

### PR DESCRIPTION
**Pull request by PixelSergey**
Also restructured the file a lil' bit

### CODE REVIEW

**Comment by PixelSergey** [comment]
Although that's true, we should still be recommending updating outdated luma versions. Hence why I changed "will cause a blackscreen" to "may cause a blackscreen", as it will not cause problems, but get gullible users to update luma.

**Comment by FrozenChen** [comment]
Lying to users to make them update luma is a big no

File: cogs/assistance.py [hunk 1]
```diff
@@ -161,21 +161,20 @@ def check_console(self, message, channel, consoles):
         """Explains how to safely prepare for an update if you have boot9strap installed"""
         await self.simple_embed(ctx, """
                 **Is it safe to update to 11.10?**
+                **To find out your Luma3DS version, hold select on bootup and look at the top left corner of the top screen**
                 
-                **Luma3DS 9.1**
+                **Luma3DS 10.0**
```
Comment (MechanicalDragon0687) [inline cogs/assistance.py]: suggest this is changed to Luma3DS >= 9.1

File: cogs/assistance.py [hunk 2]
```diff
@@ -161,21 +161,20 @@ def check_console(self, message, channel, consoles):
...
                 You can update to 11.10 safely.
                 
-                **Luma3DS 8.0 - 9.0**
-                Follow this guide: (https://bit.ly/2Q58acr), then you can update to 11.10. Being on these Luma3DS \
-versions on 11.10 will cause a blackscreen until you update.
+                **Luma3DS 8.0 - 9.1**
```
Comment (MechanicalDragon0687) [inline cogs/assistance.py]: suggest this is changed back to Luma3DS 8.0 - 9.0

**Comment by FrozenChen** [comment]
For the time being we will be using luma 9.1

**Pull request closed without merging**